jormungandr/src/jrpc/mod.rs

```diff
@@ -0,0 +1,29 @@
+#[cfg(feature = "evm")]
+mod eth_block_info;
+#[cfg(feature = "evm")]
+mod eth_types;
+
+use crate::context::ContextLock;
+use jsonrpsee_http_server::{HttpServerBuilder, RpcModule};
+use std::net::SocketAddr;
+
+pub struct Config {
+    pub listen: SocketAddr,
+}
+
+pub async fn start_jrpc_server(config: Config, _context: ContextLock) {
+    let server = HttpServerBuilder::default()
```

Comment:
nit: we may not even start the server if the evm feature is not enabled

Reply:
I have considered that jsrpc service later can be used for the original jormungandr, but agree at this moment we can disable without evm feature